composer.json

```diff
@@ -7,7 +7,7 @@
 	"require": {
 		"cakephp/cakephp": "~3.0",
 		"cakephp/app": "~3.0",
-		"phpunit/phpunit": "*",
+		"phpunit/phpunit": "^5.7.14|^6.0",
```

Comment:
Wouldnt that be in require-dev?
Or does this tool need it for upgrading?
Afaik it doesnt.

Reply:
I'm not sure, you added it here in https://github.com/cakephp/upgrade/pull/82 😉

Reply:
So maybe it was needed then.

Reply:
`phpunit` needed by `Cake\TestSuite\*` classes